Makefile

```diff
@@ -50,3 +64,10 @@ uninstall:
 		(cd $${dir} && rm -Rf node_modules); \
   	done
 .PHONY: uninstall
+
+update-functions:
+	faas-cli remove -f functions.yml
+	faas-cli up -f functions.yml --tag sha \
```

Comment:
nice, I didn't know you could do multiple commands in a make file step with using && and chaining.

Reply:
Yeah. It's similar (although not identical) to bash - the for loop above in that file highlights one of the main differences between Make and bash